bears/rest/RSTcheckBear.py

```diff
@@ -32,16 +33,32 @@ class RSTcheckBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
-                         code_block_language_ignore: list=()):
+                         code_block_language_ignore: typed_list(str)=(),
+                         directive_ignore: typed_list(str)=(),
+                         role_ignore: typed_list(str)=()):
         """
         :param code_block_language_ignore:
             Comma seperated list for languages of which code blocks
             should be ignored. Code block of following languages
             can be detected: ``bash``, ``c``, ``cpp``, ``json``,
             ``python``, ``rst``.
+
+        :param directive_ignore:
+             Comma separated list of directives to be ignored. Can be
+             used to ignore custom directives as well.
+
```

Comment:
please remove those newlines to conform to the other docstrings in our codebase :)

Reply:
removed.